publish_rate: 200 # Hz, at which everything is republished from node
base_frame_id: "odom" # Frame used as base for the teleop goal poses

# Topics
teleop_left_wrench_topic: "/teleop/left/wrench" # Where to sub to the leader's wrench
teleop_right_wrench_topic: "/teleop/right/wrench" 
teleop_gripper_left_topic: "/teleop/left/gripper" # Where to pub gripper commands
teleop_gripper_right_topic: "/teleop/right/gripper"

left_gripper_limits: [-0.5, 1.1]
right_gripper_limits: [-3.1, 1.0]

# Timeouts
pose_expiration: 0.01 # s, if a dvrk pose is this stale we do not publish
wrench_expiration: 0.1 # s, if a wrench is this stale we do not send
